src/olympia/reviewers/utils.py

```diff
@@ -244,14 +245,14 @@ def _get_addon_name_url(self, record):
 class ScannersReviewTable(AutoApprovedTable):
     listed_text = _('Listed versions needing human review ({0})')
     unlisted_text = _('Unlisted versions needing human review ({0})')
-    filters = {'needs_human_review': True}
+    filter_listed = filter_unlisted = Q(needs_human_review=True)
```

Comment:
hah, all the queries are single items in the end so we don't even need Q objects (having different `filters` dicts for listed and unlisted for `MadReviewTable` is enough).